src/Root.tsx

```diff
@@ -155,6 +158,12 @@ export const Root: FC = () => {
         title: 'Tooltips',
         href: '/tooltips',
       },
+      {
+        group: false,
+        icon: AiOutlineMinus,
+        title: 'Progressbar',
+        href: '/progressbar',
```

Comment:
This can be either `progress` or `progress-bar`

Reply:
done